src/features/homePage/HomePage.tsx

```diff
@@ -1,5 +1,115 @@
+import * as Yup from 'yup';
+import { ErrorMessage, Field, Form, Formik } from 'formik';
+import { useTranslation } from 'react-i18next';
+
+import group from '../../assets/groupLogo.png';
+import { mock } from '../products/productSeries/mockSeries';
+
 const HomePage = () => {
-  return <h1 className="text-6xl p-8 text-center">Landing Page</h1>;
+  const { t } = useTranslation('home');
+  const contactSchema = Yup.object().shape({
+    email: Yup.string().email(t('emailError')).required(t('emailError2')),
+    message: Yup.string().min(50, t('message')).required(t('message2')),
+  });
+  return (
+    <>
+      <div className="hero relative mx-auto  mb-[15.6rem] flex max-w-7xl flex-col items-end bg-no-repeat pr-8 text-right text-white">
+        <p className="mb-2.5 mt-16">S.MALLETS</p>
+        <h1 className="max-w-sm text-6xl font-bold leading-[4.8rem]">
+          Premium quality / fair price
+        </h1>
+
+        <div className="absolute inset-x-0 -bottom-40 mx-auto flex h-60 max-w-[80%] justify-center rounded-lg bg-[#EDEDED] p-10 text-left text-black sm:justify-start sm:pl-[4.4rem] lg:max-w-[58.7rem]">
+          <blockquote>
+            <p className="mb-5 max-w-[17rem]">{t('opinion')}</p>
+            <p className="font-bold">Klemens Wariat</p>
+          </blockquote>
+        </div>
+
+        <img
+          src={group}
+          alt="decoration logo"
+          className="absolute right-[2.8rem] -bottom-[15.6rem] z-50 hidden  xl:block"
+          aria-hidden="true"
+        />
+      </div>
+
+      <div>
+        <h2 className="ml-6 mb-9 text-2xl sm:mb-0">{t('subtitle')}</h2>
+      </div>
+      <div className=" relative mb-[100px] flex max-w-7xl flex-wrap justify-center">
+        {mock
+          .sort(() => 0.5 - Math.random())
+          .slice(0, 6)
+          .map((index) => (
+            <div
```

Comment:
I guess this whole "card" shuold be a link to the series page

Reply:
Hover effects more than welcome :)

Reply:
done ✅ & done ✅ (but I'm not Ala :P )